tmc-plugin/src/fi/helsinki/cs/tmc/runners/MavenExerciseRunner.java

```diff
@@ -82,16 +90,11 @@ public Integer call() throws Exception {
                 .setIO(inOut)
```

Comment:
This method is now calling getIoTab() in the Callable. Get inOut as a parameter instead.

(gah, github won't let me comment on the correct line..)